color-identifiers-mode.el

```diff
@@ -86,6 +86,13 @@ across buffers."
           (const :tag "Sequential" sequential)
           (const :tag "Hash-based" hash)))
 
+
+(defcustom color-identifiers-avoid-faces nil
```

Comment:
It would be good to add, e.g. `font-lock-warning-face` to the default value.

Reply:
@alphapapa but then I don't think most users would need to avoid warning and error faces, because not α) not everyone uses checkers, and β) even those who do use not necessarily are using ones that unable to highlight errors/warnings with an underling instead of coloring whole sentence *(which IMO is a bad design)*.

Reply:
@Hi-Angel That face is not used only in checkers.  e.g. in `emacs-lisp-mode` it is used to highlight `error`, `warn`, etc.  This mode should definitely not use that face's color.

Reply:
Well, it seems a shame to have this feature, but have it do nothing by default, and require users to understand Emacs font-locking internals to configure it properly.  Up to you, though.

Reply:
@alphapapa the problem is that two colors *(warning and error)* are too much, and then someone might come up with another favorite face which color they wouldn't want to include. I'd rather have by default as many colors as possible, because there's not many of them.

I'd vote for describing this feature in README.md instead, so those who'd like to exclude a color would know how to do it.

Reply:
> the problem is that two colors (warning and error) are too much

I don't understand what you mean.

Reply:
@alphapapa I don't know the defaults of `emacs-lisp-mode`, but "warnings" are usually drawn in yellow, and "errors" in red.

Reply:
@Hi-Angel I don't understand how that's relevant to adding `font-lock-warning-face` to this option.  There is no `font-lock-error-face`.  I'm talking about adding *one default face* to this option so it will be useful by default, and so users won't have to learn how Emacs font-locking works to configure it usefully.  Please study a bit about font-locking before offering further criticism of the suggestion.